client/me/notification-settings/reader-subscriptions/index.jsx

```diff
@@ -251,8 +282,16 @@ class NotificationSubscriptions extends Component {
 	}
 }
 
+const mapStateToProps = ( state ) => ( {
```

Comment:
Uh, I wish this was a functional component 😮‍💨

Reply:
Had the same thoughts while working on this! Let me refactor in a separate PR :)

Reply:
Resolving and moving discussion [here](https://github.com/Automattic/wp-calypso/pull/95627)